docs/project-contributions.md

```diff
@@ -2,9 +2,51 @@
 
 The information in this page is primarily for those who wish to contribute to the c2pa-python library project itself, rather than those who simply wish to use it in an application.  For general contribution guidelines, see [CONTRIBUTING.md](../CONTRIBUTING.md).
 
-## Development
+## Setup
 
-It is best to [set up a virtual environment](https://virtualenv.pypa.io/en/latest/installation.html) for development and testing.
+It is best to [set up a virtual environment](https://virtualenv.pypa.io/en/latest/installation.html) for development and testing:
+
+```bash
+# Create virtual environment
+python -m venv .venv
+```
+
+Activate the virtual environment.
+
+- On Windows:
+    ```bash
+    .venv\Scripts\activate
+    ```
+- On macOS/Linux:
+    ```bash
+    source .venv/bin/activate
+    ```
+
+Load project dependencies:
+
+```bash
+pip install -r requirements.txt
+pip install -r requirements-dev.txt
+```
+
+Download library artifacts for the current version you want, (for example, as shown below for v0.55.0):
+
+```bash
+python scripts/download_artifacts.py c2pa-v0.55.0
+```
+
+Install the package in development mode:
+
+```bash
+pip install -e .
+```
+
+This command:
+
+- Copies the appropriate libraries for your platform from `artifacts/` to `src/c2pa/libs/`
+- Installs the package in development mode, so you can make changes to the Python code without reinstalling.
+
+## Build from source
```

Comment:
Is this section still needed?

Reply:
No, it isn't anymore, good catch.

If you feel something should be mentioned for building everything yourself, it has been replaced by running https://github.com/contentauth/c2pa-python/blob/main/Makefile#L20 followed by https://github.com/contentauth/c2pa-python/blob/main/Makefile#L100 and then https://github.com/contentauth/c2pa-python/blob/main/Makefile#L26. But the shortcut command is mentioned in another part of the docs already, the one you're moving around.